ISM43362Interface.cpp

```diff
@@ -18,6 +18,8 @@
 #include "ISM43362Interface.h"
 #include "mbed_debug.h"
 
+#define LATEST_FW_VERSION_NUMBER 3525
```

Comment:
Please keep the version number in its complete origin format : e.g. C3.5.2.5-STM ...
then use ParseNumber as well when you need to compare with another version

Reply:
Done